<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Madde::Internal::MaemoDeployConfigurationWidget</class>
 <widget class="QWidget" name="Madde::Internal::MaemoDeployConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="addDesktopFileButton">
     <property name="text">
      <string>Add Desktop File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addIconButton">
     <property name="text">
      <string>Add Launcher Icon...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
